alf/algorithms/predictive_representation_learner.py

```diff
@@ -347,9 +386,13 @@ def preprocess_experience(self, experience: Experience):
             # [B, T, unroll_steps+1]
             positions = torch.min(positions, episode_end_positions)
 
-            # [B, T, unroll_steps+1]
+            # [B, T, unroll_steps+1, ...], for the scalar case, the shape is
+            # [B, T, unroll_steps+1, 1]
```

Comment:
I believe the the shape is still [B, T, unroll_steps+1] for scalar case.

Reply:
My comment was incorrect. I intended to use ``[B, T, unroll_steps+1, 1]`` do denote the desired shape.